recipes/metatensor/meta.yaml

```diff
@@ -0,0 +1,62 @@
+{% set version = "0.1.5" %}
+{% set url_base = "https://github.com/lab-cosmo/metatensor/releases/download" %}
+
+{% set CMAKE_INSTALL_PREFIX = "-DCMAKE_INSTALL_PREFIX=$PREFIX" %}  # [not win]
+{% set CMAKE_INSTALL_PREFIX = "-DCMAKE_INSTALL_PREFIX=%PREFIX%/Library" %}  # [win]
+
+{% set RUST_TARGET = "-DRUST_BUILD_TARGET=x86_64-apple-darwin" %}  # [osx and x86_64]
```

Comment:
Please use `-DRUST_BUILD_TARGET=$CONDA_RUST_TARGET` instead of manually listing them.

Reply:
thanks, I was looking for something like this!

Reply:
Hmm, it looks like `$CONDA_RUST_TARGET` is not using the right format: it is `X86_64_UNKNOWN_LINUX_GNU` instead of `x86_64-unknown-linux-gnu` on linux for example. Is there another variable I can use instead?

Reply:
`CARGO_BUILD_TARGET` is the one then. See https://github.com/conda-forge/rust-activation-feedstock/blob/77337e3832b712a6de383c0a96dbe1a33d5f864f/recipe/activate.sh#L9-L16 and https://github.com/conda-forge/rust-activation-feedstock/blob/77337e3832b712a6de383c0a96dbe1a33d5f864f/recipe/conda_build_config.yaml for the source of these values.